apps/api/package.json

```
@@ -25,7 +25,8 @@
     "test:e2e:ee": "cross-env TS_NODE_COMPILER_OPTIONS='{\"strictNullChecks\": false}' TZ=UTC NODE_ENV=test E2E_RUNNER=true mocha --timeout 10000 --require ts-node/register --exit --file e2e/setup.ts src/**/*.e2e-ee.ts",
     "migration": "cross-env NODE_ENV=local MIGRATION=true ts-node --transpileOnly ./migrations/expire-at/expire-at.migration.ts",
     "migration:in-app": "cross-env NODE_ENV=local MIGRATION=true ts-node --transpileOnly ./migrations/integration-scheme-update/add-primary-priority-migration.ts",
-    "migration:primary-provider": "cross-env NODE_ENV=local MIGRATION=true ts-node --transpileOnly ./migrations/integration-scheme-update/add-primary-priority-migration.ts"
+    "migration:primary-provider": "cross-env NODE_ENV=local MIGRATION=true ts-node --transpileOnly ./migrations/integration-scheme-update/add-primary-priority-migration.ts",
+    "link:submodules": "cd ../../enterprise/packages/packages/auth && yarn link && cd ../../../../apps/api && yarn link @novu/ee-auth"
```

similar idea about the separate script that will do the linking

---

```suggestion
    "link:submodules": "cd ../../enterprise/packages/packages/auth && yarn link && cd ../../../../apps/api && pnpm link @novu/ee-auth"`
```
please change to `pnpm link`, but check the docs: https://pnpm.io/cli/link